sematic/db/queries.py

```diff
@@ -158,22 +158,24 @@ def get_run_status_details(
     jobs for the run (if any exist)
     """
     with db().get_session() as session:
-        query_results = (
-            session.query(Run.id, Run.future_state, Run.external_jobs_json)
+        query_results = list(
+            session.query(Run.id, Run.future_state, Job)
+            .outerjoin(Job, Job.run_id == Run.id, full=True)
             .filter(Run.id.in_(run_ids))
+            .filter(sqlalchemy.or_(Job.kind == JobKind.run, Job.kind.is_(None)))
```

Comment:
`Job.run_id` must be made `Optional[str]` and `nullable=True`.

Reply:
@tscurtu  When would `job.run_id` be null?

@augray when does `Job.kind.is_(None)` occur?